app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import T from 'i18n-react';
+import {
+  Grid,
+  GridHeader,
+  GridBody,
+  GridRow,
+  GridCell,
+  StyledIcon,
+  LinedSpan,
+} from '../shared.styles';
+import { ICONS, CONNECTIONS_TABLE_HEADERS, PREFIX, DESC_COLUMN_TEMPLATE } from './constants';
+import { IConnection, ITableData } from '../types';
+import { humanReadableDate } from 'services/helpers';
+
+const COLUMN_TEMPLATE = '50px 200px 2fr 1.5fr 1.5fr 200px 1.5fr 120px 120px 225px';
+const REQUEST_DATE_FORMAT = 'MM/DD/YYYY - hh:mm A';
```

Comment:
Please use `DataFormatter`

Reply:
Addressed in [2bcf545f](https://github.com/cdapio/cdap-ui/pull/354/commits/2bcf545f7321480e53d8aea16fa53180404c28d4)